src/jquery.multiselect.js

```diff
@@ -865,6 +1013,8 @@
     /**
      * Toggles the checked state on options within the menu
      * Potentially scoped down to visible elements from filteredInputs
+     * Groups don't show up in the filter search right now, so both
```

Comment:
I deleted this portion of the comment intentionally, it wasn't true.

Reply:
Oh, did I revert the delete by accident?